.appveyor.yml

```diff
@@ -38,6 +39,8 @@ install:
   - npm install -g npm@%NPM_VERSION%
   - npm install -g bower@%BOWER_VERSION% yarn@%YARN_VERSION%
   - pip install virtualenv==%VIRTUAL_ENV_VERSION%
+  - pip install conan==%CONAN_VERSION%
+  - cmd: conan user # Create the conan data directory. Automatic detection of your arch, compiler, etc.
```

Comment:
Just out of curiosity: Did you actually try without the `cmd:`? As I believe that should work, too.